server/lib/src/be/idl_sqlite.rs

```diff
@@ -1771,7 +1771,6 @@ impl IdlSqlite {
                         "PRAGMA page_size={fs_page_size};
                          PRAGMA cache_size={cache_pages};
                          PRAGMA journal_mode=WAL;
-                         PRAGMA synchronous=NORMAL;
```

Comment:
What's the default/setting in this case? the [sqlite docs](https://www.sqlite.org/pragma.html#pragma_synchronous) don't seem to mention it? 🫨

Reply:
It's "synchronous=FULL" but you can relax it to NORMAL if you have WAL. I think I added it earlier, but I reliased now it's not a good idea to set in that layer as it doesn't make much performance change, and an admin can always just do it direct into the underlynig sqlite file if they want to change it.

Reply:
Ah, OK. they do mention that "NORMAL" is normally the best case in WAL mode, but more sync more better.